packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -688,3 +690,65 @@ export function isParentOf(element, targetElement) {
     false
   );
 }
+
+/**
+ * Copies text to clipboard using the Clipboard API.
+ * https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
+ * @param {string} text 
+ */
+export async function copyToClipboard(text) {
+  if (navigator?.clipboard?.writeText) {
+    try {
+      await navigator.clipboard.writeText(text);
+    } catch (err) {
+      copyToClipboardFallback(text);
+    }
+  } else {
+    copyToClipboardFallback(text);
+  }
+}
+
+/**
+ * Copies text to clipboard using the DOM. 
+ * @param {string} text
+ */
+export function copyToClipboardFallback(text) {
+  const el = document.createElement('textarea');
+  el.value = text;
+  document.body.appendChild(el);
+  el.select();
+  document.execCommand('copy');
+  document.body.removeChild(el);
+}
+
+/**
+ * Read the value of a given key from an object.
+ * @param {object} object 
+ * @param {string} key 
+ */
+export function readFromObject(object, key) {
+  const firstPeriodIndex = key.indexOf('.');
+  if (object && firstPeriodIndex !== -1) {
+    let newKey = key.substring(firstPeriodIndex + 1);
+    key = key.substring(0, firstPeriodIndex);
+    return readFromObject(object[key], newKey);
+  }
+  return object ? object[key] : undefined;
+}
+
+/**
+ * Generates a text from a result based on a given template.
+ * @param {string} template 
+ * @param {Result} result 
+ * @returns {string}
+ */
+export function buildTemplateTextFromResult(template, result) {
```

Comment:
Can you please let us know why this is there and not in the copy to clipboard file?
Is there another place where you would be using that?

Reply:
Yes sure, actually I found this function very useful and it can help with multiple result actions. For example the Post To Feed and the Send As Email  actions will use it as well to generate a text from the result to be set in the feed or in the Email.
That's why I think it would be better in the QuanticUtils so it can be easily re-used by us in SFINT and by any customer who wants to create his own result actions.

Reply:
Gotcha makes sense!